### PR TITLE
Update definitions for react-onclickoutside 6

### PR DESCRIPTION
**Pull request by LKay**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [X] Increase the version number in the header if appropriate.
- [X] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by mhegazy** [comment]
@andy-ms this looks like a definition tester bug. can you take a look.

File: types/react-onclickoutside/v5/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "../tsconfig.json",
```
Comment (andy-ms) [inline types/react-onclickoutside/v5/tsconfig.json]: We don't support `extends`, please leave this file as it was before.

File: types/react-onclickoutside/v5/tslint.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../tslint.json"
```
Comment (andy-ms) [inline types/react-onclickoutside/v5/tslint.json]: Change to `"extends": "dtslint/dt.json"`.

**Comment by andy-ms** [comment]
Thanks!

**Pull request merged**